collector/lambdacomponents/go.mod

```diff
@@ -2,4 +2,13 @@ module github.com/open-telemetry/opentelemetry-lambda/collector/lambdacomponents
 
 go 1.16
```

Comment:
Think this file's go.sum isn't updated yet

Reply:
Should be good now.